pipeflow_lib.tlv

```diff
@@ -934,9 +933,23 @@ m4_popdef(['m4_trans_ind'])
             $maybe_valid = /_top|_in_pipe<>0$valid || $delayed;
             ?$maybe_valid
                /_trans_hier
- m4_trans_ind              $ANY = $delayed ? /_top|_in_pipe/trans>>1$ANY : /_top|_in_pipe/pred_trans<>0$ANY;
+            m4_trans_ind   $ANY = $delayed ? /_top|_in_pipe/trans>>1$ANY : /_top|_in_pipe/pred_trans<>0$ANY;
          @_comp_stage
             $valid = (/_top|_in_pipe<>0$valid && ! /_top|_in_pipe/comp<>0$mispred) || $delayed;
    m4_popdef(['m4_trans_ind'])
 
-
+\TLV arb(/_top_in1,/_top_in2,|_in1,|_in2,|_arb_out)
```

Comment:
This macro is still specific to the flow example. We want a macro that is generic. Please study the flow-control mechanism that is used, and consider what is required to make it generic and consistent w/ the interface to other flow macros. Please write the comment header for the macro before writing the macro, and we can review that together.